udata_gouvfr/theme/gouvfr/templates/security/change_password.html

```diff
@@ -1,25 +1,25 @@
-{% extends theme('layouts/1-column.html') %}
-{% import theme('macros/forms.html') as forms with context %}
+{% extends theme('base.html') %}
+{% import theme('macros/forms2.html') as forms with context %}
```

Comment:
Looks like it's used only in the `security/` templates. I think we should rename this to `forms.html` and rename the previous one to `forms_old.html` to keep history while we are working on this and be consistent with the other templates we've renamed.